polyfills/Array/prototype/find/config.toml

```
@@ -15,16 +15,16 @@ docs = "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global
 [browsers]
 bb = "*"
 chrome = "<=45"
-edge = "*"
-edge_mob = "*"
+edge = "<12"
+edge_mob = "<12"
 firefox = "<24"
 ie = "*"
 ie_mob = "10 - *"
 opera = "<32"
-safari = "* - 7.1"
-ios_saf = "* - 8.0"
-ios_chr = "* - 8.0"
-android = "*"
+safari = "<7.1"
```

```suggestion
safari = "<8.0"
```

Same as mdn says and`ios_saf`, et?

---

thank you for the spot, I refactored this incorrectly 🙈